docs/index.md

```diff
@@ -467,7 +467,11 @@ No. SMART Health Cards are designed for use *alongside* existing forms of identi
 ## How can we share conclusions like a "Safe-to-fly Pass", instead of sharing clinical results?
 Decision-making often results in a narrowly-scoped "Pass" that embodies conclusions like "Person X qualifies for international flight between Country A and Country B, according to Rule Set C". While Health Cards are designed to be long-lived and general-purpose, Passes are highly contextual. We are not attempting to standardize "Passes" in this framework, but Health Cards can provide an important verifiable input for the generation of Passes.
 
+## How there tools to validate implementations of the framework.
```

Comment:
```suggestion
## What testing tools are available to validate SMART Health Cards implementations?
```

Reply:
Wow, my French Friday afternoon brain went all out on the homonyms train there :)